providers/implementations/storemgmt/file_store.c

```diff
@@ -511,7 +487,6 @@ static int file_setup_decoders(struct file_ctx_st *ctx)
 
     ok = 1;
  err:
-    OSSL_DECODER_free(to_obj);
```

Comment:
This is wrong. The to_obj decoder is up-refed by ossl_decoder_instance_new(). This is the cause of the sanitizer CI test failure.

Reply:
Well, uhm, dropping that *here* ***is*** correct, since `to_obj` is now within the decode addition loop